test/lib/specHelperSpec.js

```diff
@@ -92,21 +92,21 @@ describe("Fluxxed up test helpers", function() {
 
   describe("Test Rig", function() {
     var rig = new TestRig()
-    beforeEach(()=>{
-      rig.boltOn(TestComponent)
+    beforeEach(() => {
```

Comment:
@maneeshnycda Thanks for following my syntax preference despite me having no good argument for why I want it to be formatted that way.